bootstrap/terraform/README.md

```diff
@@ -90,6 +90,10 @@ kubectl get nodes
 ```shell script
 kubectl get pods -n crossplane-system
 ```
+#### Step8: Verify the names of the providerConfig resources for all providers installed
+```shell script
+kubectl get provider
```

Comment:
This gets Crossplane providers, not ProviderConfigs.

Reply:
@nabuskey there is a difference between `kubectl get provider` and `kubectl ger providers`

The command `kubectl get provider` will returns all CRs that the corresponding CRD `category` has `provider` like you can see in the kubernetes providerconfig CRD https://github.com/crossplane-contrib/provider-kubernetes/blob/main/package/crds/kubernetes.crossplane.io_providerconfigs.yaml#L14

I added both commands in the instructions